nimare/workflows/scale.py

```diff
@@ -22,17 +23,22 @@
                     'frequency bias into account, for delineating distinct core networks of '
                     'coactivation, using a permutation based approach with Family Wise Error '
                     'multiple comparison correction.')
-                    
+
 @click.argument('database', required=True, type=click.Path(exists=True, readable=True))
+@click.option('--base_img', type=click.Path(exists=True, readable=True), help='Voxelwise baseline activation rates.')
```

Comment:
Can you change this to `baseline`? Since we support text files, and will support images at some point, I think a more general argument is better.